### PR TITLE
 LTD-4129-inform-letter-status-decision-docs

### PR DESCRIPTION
**Pull request by depsiatwal**
Advice documents drive many of the document generation. 

Inform has been added to advice_type as it's required by login in the FE. 
With out this too many hacks are required for figure out if a document or template is of type inform

### CODE REVIEW

File: api/cases/libraries/finalise.py [hunk 1]
```diff
@@ -19,6 +19,10 @@ def get_required_decision_document_types(case):
         required_decisions.add(AdviceType.APPROVE)
         required_decisions.remove(AdviceType.PROVISO)
 
+    # Ensure that REFUSE advice requires an inform document
+    if AdviceType.REFUSE in required_decisions:
+        required_decisions.add(AdviceType.INFORM)
```
Comment (currycoder) [inline api/cases/libraries/finalise.py:24]: Do we want to put this behind the feature flag too?

**Review by currycoder — COMMENTED** [review]
Looks good, just one comment

File: api/cases/libraries/finalise.py (hunk 1 above)
Comment (depsiatwal) [reply api/cases/libraries/finalise.py:24]: We can put we have in FF in FE for decision documents
also we will still need to block inform letter from the finalisation screen anyway so don't see it adding value. 

But having said that lets do it to be safer

**Review by seijihg — APPROVED** [review]

File: api/cases/libraries/finalise.py (hunk 1 above)
Comment (depsiatwal) [reply api/cases/libraries/finalise.py]: add with tests

**Review by currycoder — APPROVED** [review]

**Pull request merged**